diff --git a/dist/metro/index.js b/dist/metro/index.js
index c91ba6a..61ea5c2 100644
--- a/dist/metro/index.js
+++ b/dist/metro/index.js
@@ -177,17 +177,20 @@ async function startCSSProcessor(filePath, platform, isDev, { input, getCSSForPl
                 : getNativeJS((0, css_to_rn_1.cssToReactNativeRuntime)(css, options), debug)));
             debug(`virtualStyles.emit ${platform}`);
             haste.emit("change", {
-                eventsQueue: [
-                    {
-                        filePath,
-                        metadata: {
-                            modifiedTime: Date.now(),
-                            size: 1,
-                            type: "virtual",
-                        },
-                        type: "change",
-                    },
-                ],
+                changes: {
+                    addedFiles: new Map(),
+                    modifiedFiles: new Map([
+                        [
+                            path_1.default.relative(process.cwd(), filePath),
+                            {
+                                isSymlink: false,
+                                modifiedTime: Date.now(),
+                            },
+                        ],
+                    ]),
+                    removedFiles: new Map(),
+                },
+                rootDir: process.cwd(),
             });
         }).then((css) => {
             debug(`virtualStyles.initial ${platform}`);
@@ -277,4 +280,4 @@ function stringify(data) {
         }
     }
 }
-//# sourceMappingURL=index.js.map
\ No newline at end of file
+//# sourceMappingURL=index.js.map
diff --git a/src/metro/index.ts b/src/metro/index.ts
index fa58c7b..9ad0147 100644
--- a/src/metro/index.ts
+++ b/src/metro/index.ts
@@ -342,17 +342,20 @@ async function startCSSProcessor(
 
         debug(`virtualStyles.emit ${platform}`);
         haste.emit("change", {
-          eventsQueue: [
-            {
-              filePath,
-              metadata: {
-                modifiedTime: Date.now(),
-                size: 1, // Can be anything
-                type: "virtual", // Can be anything
-              },
-              type: "change",
-            },
-          ],
+          changes: {
+            addedFiles: new Map(),
+            modifiedFiles: new Map([
+              [
+                path.relative(process.cwd(), filePath),
+                {
+                  isSymlink: false,
+                  modifiedTime: Date.now(),
+                },
+              ],
+            ]),
+            removedFiles: new Map(),
+          },
+          rootDir: process.cwd(),
         });
       }).then((css) => {
         debug(`virtualStyles.initial ${platform}`);
